docs/tutorial.rst

```diff
@@ -949,18 +949,18 @@ and ``diff_vis`` methods.
 
 UVData: Working with large files
 --------------------------------
-To save on memory and time, pyuvdata supports reading only parts of uvfits, uvh5 and
-miriad files.
+To save on memory and time, pyuvdata supports reading only parts of uvfits, uvh5,
+miriad, and MWA correlator fits files.
 
-a) Reading the metadata of a uvfits, uvh5 or miriad file
+a) Reading the metadata of a uvfits, uvh5, miriad, or mwa corr fits file
 ********************************************************
-For uvh5 and uvfits files, reading in the metadata results in a metadata only
+For uvh5, uvfits, miriad, and MWA correlator fits files, reading in the metadata results in a metadata only
```

Comment:
I'm don't think Miriad should be listed here because a miriad metadata only read doesn't result in a metadata only object that passes checks -- it's missing many required attributes even for a metadata only object. The details around Miriad are in the next sentence.